front/src/applications/operationalStudies/types.ts

```diff
@@ -18,6 +18,7 @@ import type { SuggestedOP } from 'modules/timetableItem/types';
 import type { Train, TimetableItemWithPathOps } from 'reducers/osrdconf/types';
 import type { Duration } from 'utils/duration';
 import type { ArrayElement } from 'utils/types';
+import type { NodeIndexed } from './views/Scenario/components/MacroEditor/MacroEditorState';
```

Comment:
I don't think we should import anything from `MacroEditor` in the import code, and especially not the `MacroEditorState` which does a lot more than is necessary.

Fetching existing macro nodes without `MacroEditor` functions should be pretty easy I think?

Reply:
Ah, since it's just a standalone helper it's probably fine. I think ideally it should be in `osrdEditoastApi` (we already have some other `*All*` helpers in there).